app/grants/views.py

```diff
@@ -332,11 +332,11 @@ def bulk_grants_for_cart(request):
     return JsonResponse({'grants': grants})
 
 
-def clr_grants(request, round_num):
+def clr_grants(request, round_num, sub_round_slug=''):
     """CLR grants explorer."""
 
     try:
-        clr_round = GrantCLR.objects.get(round_num__icontains=round_num)
+        clr_round = GrantCLR.objects.get(round_num=round_num, sub_round_slug=sub_round_slug)
```

Comment:
if sub_round_slug is empty -> you would get more than one result!

Reply:
fixed.